### ISSUE TITLE
Track Editor: cannot set default settings of rail logics

### ISSUE THREAD

**Issue by DasCapschen** [Libre-TrainSim/Libre-TrainSim#408]
### Summary
When adding or selecting a rail logic, you can select the visual instance and where to place it, but you cannot modify its settings.  
For example, Signals and Contact Points should have options for speed limits.

I believe you can set speed and such for signals per Scenario, but that's kind of annoying, because you have to do every signal every time, instead of having default settings to override.

### How to reproduce
1. open editor
2. add a rail logic to a rail (or select one)
3. go to rail logic settings tab
4. no options :(

![grafik](https://user-images.githubusercontent.com/3923406/189528098-4fb360bd-50cb-44b3-a3a7-4af7acd7ebe4.png)

**Comment by HaSa1002** [comment]
Yeah. Jean described this behaviour as feature. I agree the UX is not nice, though
